wagtail_wordpress_import/test/tests/test_shortcode_handlers.py

```diff
@@ -291,16 +354,140 @@ def test_construct_block_method_output(self):
         json = handler.construct_block(
             wagtail_custom_html.find("wagtail_block_caption")
         )
+        image = get_image_model().objects.get(title="j-money-family-portrait.jpg")
+
         self.assertIsInstance(json, dict)
         self.assertEqual(json["type"], "image")
-        # ["value"]["image"] is an image id. it should be returned as an integer but we cannot
-        # reply on the value returned here so check if it is an integer.
-        # there is a ticket to improve testing when fetching remote images:
-        # https://projects.torchbox.com/projects/wordpress-to-wagtail-importer-package/tickets/76
-        self.assertIsInstance(int(json["value"]["image"]), int)
+        self.assertEqual(json["value"]["image"], image.id)
```

Comment:
**Question**: Wasn't this also fixed in #98?

Reply:
#98 was for the test_block_builder TODO